### PR TITLE
feat:  Add NLP Engine

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <GordonJSmith@gmail.com>

### CODE REVIEW

File: overlays/nlp-engine/vcpkg.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "nlp-engine",
+  "version-semver": "1.21.7",
```
Comment (dehilsterlexis) [inline overlays/nlp-engine/vcpkg.json:3]: @GordonSmith is this the NLP Engine version? If so, it would need to be updated.

**Review by dehilsterlexis — CHANGES_REQUESTED** [review]
Just a comment on a version number. Not sure it pertains to the NLP Engine.

File: overlays/nlp-engine/vcpkg.json (hunk 1 above)
Comment (dehilsterlexis) [reply overlays/nlp-engine/vcpkg.json:3]: Yes.

**Review by dehilsterlexis — APPROVED** [review]

**Comment by GordonSmith** [comment]
Fixed building on OSX

**Pull request merged**